src/helpers/CBV/SingleObject.tsx

```diff
@@ -0,0 +1,74 @@
+import { Registry } from '@onaio/redux-reducer-registry';
+import React from 'react';
+import { ActionCreator } from 'redux';
+import { Base } from './Base';
+
+/** interface for the props of the connected component created by SingleObject  */
+interface SingleObjectProps<TAction, TObject> {
+  actionCreator: ActionCreator<TAction>;
+  object?: TObject;
+}
+
+/**
+ * SingleObject class
+ *
+ * This class is initialized with the following parameters:
+ *    - component: a react component
+ *    - options: an object representing options for SingleObject
+ *
+ * The class's render method will then return the same component wrapped in a
+ * higher order component that has been connected to the redux store, and which
+ * knows how to fetch data for display.
+ *
+ * The end goal is a quick and convenient way to display a singular objects without
+ * having to worry about boilerplate code that deal with fetching it and
+ * storing it in a Redux store.
+ *
+ * Every method in this class can and should be overwritten to cater to custom needs.
+ */
+// export class SingleObject<ObjectType, ActionType, SelectorType, PropsType, RootState = Registry> {
+export class SingleObject<
```

Comment:
@cKellyDesign @p-netm we now have `SingleObject` that is concerned with components that only want one object from the store.